erpnext/manufacturing/doctype/production_planning_tool/production_planning_tool.json

```diff
@@ -235,10 +288,87 @@
    "in_filter": 0, 
    "in_list_view": 0, 
    "label": "Sales Orders", 
+   "length": 0, 
    "no_copy": 0, 
    "options": "Production Plan Sales Order", 
    "permlevel": 0, 
    "print_hide": 0, 
+   "print_hide_if_no_value": 0, 
+   "read_only": 0, 
+   "report_hide": 0, 
+   "reqd": 0, 
+   "search_index": 0, 
+   "set_only_once": 0, 
+   "unique": 0
+  }, 
+  {
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "depends_on": "eval: doc.get_items_from == \"Material Request\"", 
+   "fieldname": "section_break_16", 
+   "fieldtype": "Section Break", 
+   "hidden": 0, 
+   "ignore_user_permissions": 0, 
+   "in_filter": 0, 
+   "in_list_view": 0, 
+   "length": 0, 
+   "no_copy": 0, 
+   "permlevel": 0, 
+   "precision": "", 
+   "print_hide": 0, 
+   "print_hide_if_no_value": 0, 
+   "read_only": 0, 
+   "report_hide": 0, 
+   "reqd": 0, 
+   "search_index": 0, 
+   "set_only_once": 0, 
+   "unique": 0
+  }, 
+  {
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "description": "Pull Material Request of type Manufacture based on the above criteria", 
+   "fieldname": "get_material_request", 
+   "fieldtype": "Button", 
+   "hidden": 0, 
+   "ignore_user_permissions": 0, 
+   "in_filter": 0, 
+   "in_list_view": 0, 
+   "label": "Get Material Request", 
+   "length": 0, 
+   "no_copy": 0, 
+   "options": "get_pending_material_requests", 
```

Comment:
method name in options is not a good idea, it becomes very difficult to manage it in the future. Suppose, you are refactoring code and renamed the method, on search and replace it will not appear.